proxy/Cargo.toml

```diff
@@ -32,14 +32,13 @@ rand = "0.4"
 # for config parsing
 regex = "1.0.0"
 
-tokio-core = "0.1"
-tokio-io = "0.1"
-tokio-signal = "0.1"
+tokio = "0.1.6"
+tokio-signal = { git = "https://github.com/alexcrichton/tokio-signal.git", rev = "7ecaa90dfe21edb1afa48c0173cbee6c3c210130"}
```

Comment:
what's the status of tokio-signal? @carllerche

Reply:
Looks like they cut a version which supports the new Tokio a few days ago: https://crates.io/crates/tokio-signal/0.2.0. Will use that.